<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioRecorder</class>
 <widget class="QMainWindow" name="AudioRecorder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Status of the server accessibility</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="6">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QComboBox" name="audioDeviceBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input Device:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="sendFileMain">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>SEND FILE</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="reciveFileMain">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string>RECIVE FILE</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" alignment="Qt::AlignLeft">
     <widget class="QPushButton" name="statusBtn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="results">
      <property name="text">
       <string>Results.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>setOutputLocation()</slot>
  <slot>toggleRecord()</slot>
  <slot>togglePause()</slot>
 </slots>
</ui>
